app/src/main/java/com/github/se/polyfit/ui/theme/Color.kt

```diff
@@ -9,3 +9,6 @@ val Pink80 = Color(0xFFEFB8C8)
 val Purple40 = Color(0xFF6650a4)
 val PurpleGrey40 = Color(0xFF625b71)
 val Pink40 = Color(0xFF7D5260)
+val BlueButton =
+    Color(0xFF7158E2) // I quickly to match the figma, I don't know which material it is.
```

Comment:
nit: is there any reason why it is not on the same line

Reply:
Nop, the comment too long and ktfmt